release-team/role-handbooks/test-infra/README.md

```diff
@@ -0,0 +1,60 @@
+# Test Infra Lead Playbook
+
+## Overview of Test Infra Lead responsibilities
+
+Note: Currently, the test-infra lead has to be someone from Google GKE Engprod Team, in order to gain access to the prow cluster. This
+will change once we migrate our testing infrastructure under CNCF account.
+
+There are two major area that test-infra lead need to take care during the release cycle, which are:
+
+1. Create CI/Presubmit jobs for the new release, and polulate the testgrid dashboard
+2. Watch for test infra status, make sure test infra is stable, and react to test infra related issues
+
+It's important to ensure the stability of our test infra during the release cycle, so that we can get reliable testing signals throughout the release cycle.
+
+You can also work with @kubernetes/test-infra-maintainers or [test infra oncall](https://go.k8s.io/oncall) if you are blocked by anything.
+
+Also feel free to ping #sig-testing and #testing-ops to reach out for help.
+
+### Create CI/Presubmit jobs for the new release
+
+This step should happen in week 6-7, when we create the new release branch.
+
+Most of the release blocking jobs are named with -beta|-stableX, which are mapped to our [release channels](https://github.com/kubernetes/test-infra#release-branch-jobs--image-validation-jobs).
+
+TODO(krzyzacy): this section reflects the status of the world today, we are actively looking for simplify the process.
```

Comment:
Did you want this line visible or wrapped in ```<!-- ... >```?

Reply:
removed the TODO as the doc is up-to-date

We can update the doc in the future if we simplified any steps.